doc/caveates.md

```diff
@@ -0,0 +1,41 @@
+Caveats
+=======
+
+Git-dit is, obviously, git specific and will hence not work with other VCS like
+mercurial. Import and export functionality for other issue tracking systems are,
```

Comment:
Wouldn't mention another VCS by name here. It does not work with others. Period.